src/chrome/__mocks__/chrome.js

```diff
@@ -34,8 +34,14 @@ export const mockChrome = (opts = {}) => {
         notificationListeners.push(callback);
       },
     },
-    clear: (id) => {
+    clear: (id, callback) => {
+      if (typeof callback !== 'function') {
+        // Older versions of chrome fail without this
+        const msg = "errorInvocation of form notifications.clear(string) doesn't match definition";
+        throw new Error(msg);
+      }
       delete currentNotifications[id];
+      setTimeout(() => { callback(true); });
```

Comment:
This is just mimicking what chrome does (it calls the callback when it's done: https://developer.chrome.com/extensions/notifications#method-clear). The `setTimeout` is just to make it a bit more realistic (so it will be async), doesn't really matter but I figured.

All versions of chrome call the callback if it's provided; earlier versions just require the callback (whereas it's optional in newer versions).